src/classes/HiFiMixerSession.ts

```diff
@@ -831,11 +790,7 @@ export class HiFiMixerSession {
                 HiFiLogger.warn(`Couldn't set input audio on _raviSession.streamController: No \`streamController\`!`);
                 retval = false;
             } else {
-                if (inputAudioMediaStream) {
-                    streamController.setInputAudio(inputAudioMediaStream.clone(), isStereo);
-                } else {
-                    streamController.setInputAudio(null, isStereo);
-                }
+                streamController.setInputAudio(inputAudioMediaStream, isStereo);
```

Comment:
Are we sure we want to lose this `clone()` behavior on the `inputAudioMediaStream`?